sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## 1.0.0-beta.3 (Unreleased)
 
+FormTraining Client updates:
+- Adopt to `training` namespace for Form Recognizer Training Clients
+- Rename parameter `fileSourceUrl` to `trainingFilesUrl` on FormTrainingClients
+- Rename parameter `useLabelFile` to `useTrainingLabels()` on FormTrainingClients
+- Add `TrainModelOptions` model for training file filter properties of `filePrefix` and `includeSubFolders`
```

Comment:
Also correct me if I'm wrong, but it looks like you use the property name `prefix` in the code, not `filePrefix`

Reply:
Oh, this is referring to the param name previously exposed in the API's and not the generated client names.

Reply:
Oh got it. Maybe say that it's replacing those parameters?

Reply:
done